gnome-initial-setup/pages/endless-eula/gis-endless-eula-page.ui

```diff
@@ -52,12 +52,11 @@
           </packing>
         </child>
         <child>
-          <object class="GtkScrolledWindow" id="eula_scrolledwin">
+          <object class="GtkAlignment" id="eula_viewer_container">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
             <property name="hexpand">True</property>
             <property name="vexpand">True</property>
-            <property name="shadow_type">in</property>
```

Comment:
Do we not like shadows any more?

Reply:
This is now a `GtkAlignment`, the shadow is applied in `GisEndlessEulaViewer`.